Calibration/IsolatedParticles/plugins/IsolatedTracksCone.cc

```diff
@@ -73,26 +348,31 @@ IsolatedTracksCone::IsolatedTracksCone(const edm::ParameterSet& iConfig) :
 
   edm::ParameterSet parameters = iConfig.getParameter<edm::ParameterSet>("TrackAssociatorParameters");
   edm::ConsumesCollector iC = consumesCollector();
-  parameters_.loadParameters( parameters, iC );
+  parameters_.loadParameters( parameters, iC);
   trackAssociator_ =  new TrackDetectorAssociator();
```

Comment:
not part of this PR, but you could use a unique_ptr for the `TrackDetectorAssociator`